src/main/java/seedu/address/logic/parser/SortCommissionCommandParser.java

```diff
@@ -22,7 +22,7 @@ public SortCommissionCommand parse(String args) throws ParseException {
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_SORT);
 
         if (argMultimap.getValue(PREFIX_SORT).isEmpty()) {
-            return new SortCommissionCommand(false, true);
+            throw new ParseException(MESSAGE_INVALID_SORT_DIRECTION);
```

Comment:
Shouldn't this be another message like missing direction or invalid format? What happens when I run just `comm`?